code/game/objects/items/granters.dm

```diff
@@ -430,3 +430,23 @@
 	icon_state = "cooking_learing_sweets"
 	oneuse = FALSE
 	remarks = list("So that is how icing is made!", "Placing fruit on top? How simple...", "Huh layering cake seems harder then this...", "This book smells like candy", "A clown must have made this page, or they forgot to spell check it before printing...", "Wait, a way to cook slime to be safe?")
+
+/obj/item/book/granter/crafting_recipe/pipegun_prime
+	name = "diary of a dead assistant"
+	desc = "A battered journal. Looks like he had a pretty rough life."
+	crafting_recipe_types = list(
+		/datum/crafting_recipe/pipegun_prime
+	)
+	icon_state = "book1"
+	oneuse = TRUE
+	remarks = list("He apparenty mastered some lost guncraftng technique.", "Why do I have to go through so many hoops to get this shitty gun?", "That much Grey Bull cannot be healthy...", "Did he drop this into a moisture trap? Yuck.", "Toolboxing techniques, huh? I kinda just want to know how to make the gun.", "What the hell does he mean by 'ancient warrior tradition'?")
```

Comment:
```suggestion
	remarks = list("He apparently mastered some lost guncrafting technique.", "Why do I have to go through so many hoops to get this shitty gun?", "That much Grey Bull cannot be healthy...", "Did he drop this into a moisture trap? Yuck.", "Toolboxing techniques, huh? I kinda just want to know how to make the gun.", "What the hell does he mean by 'ancient warrior tradition'?")
```

Reply:
nyoops I'll fix this with fixing the conflicts